package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
I fear, it's not only not documented, but also not even mentioned [in the sources](https://github.com/sindresorhus/ava/blob/master/cli.js#L57-L107) anywhere ...